application/pom.xml

```diff
@@ -261,6 +266,22 @@
             <groupId>org.hsqldb</groupId>
             <artifactId>hsqldb</artifactId>
         </dependency>
+        <dependency>
+            <groupId>eu.infomas</groupId>
+            <artifactId>annotation-detector</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.hashmap.tempus</groupId>
+            <artifactId>annotations</artifactId>
+            <version>1.0-SNAPSHOT</version>
+        </dependency>
```

Comment:
Once TempusSparkAnnotations is published on Maven central, i will update this dependency.